src/samples/annotation-list/v1/first-page.json

```diff
@@ -0,0 +1,47 @@
+{
+    "total": 4,
+    "items": [
+        {
+            "uri": "https://hyp.is/41pgvpe9EeelPR-Oxb-4UQ/elifesciences.org/inside-elife/77dd2808/taking-a-closer-look-screening-of-images",
+            "published": "2017-09-15T14:40:52Z",
+            "updated": "2017-09-15T14:42:43Z",
+            "title": "Taking a closer look: Screening of images",
+            "target": "Rossner and Yamada 2004, Cromey 2010",
+            "content": [
+                {
+                    "type": "paragraph",
+                    "text": "<strong>Bold text<strong> and <em>italic</em> with a <a href=\"https://elifesciences.org\">link</a>"
+                },
+                {
+                    "type": "quote",
+                    "text": [
+                        {
+                            "type": "paragraph",
+                            "text": "Blockquote"
+                        }
+                    ]
+                },
+                {
+                    "type": "paragraph",
+                    "text": "https://cdn.elifesciences.org/style-guide-images/elife-full-color-horizontal.png"
```

Comment:
Is `image` meant to be a possibility?

Reply:
This could be interesting. The images that are displayed in the annotations are only referenced, they must be hosted elsewhere. We only support displaying images served on a IIIF server. I will talk to @NickDuf about how we should present these on the profiles page.

Reply:
For what it's worth, it's insanely easy to add a new source to our IIIF that points to `cdn.elifesciences.org` or to somewhere similar.

Reply:
So the images aren't hosted by Hypothesis? We might want to do what we do with Medium and download them, and connect Loris to Annotations. (Preserves the record of the image too, though I guess it would change/disappear through the Hypothesis client?)

Reply:
Yes, the hypothesis client would not use our images unless we edited the annotation and that would not be appropriate. The images in an annotation are stored as url's only. They could be hosted absolutely anywhere.

My recommendation for now is that we either present them as clickable links or that we convert them to `<img src="http://example/image.png" />`. Will chat to @NickDuf.

Reply:
We already handle inline images, so that would work. But if they include something that's massive/disappears...

Reply:
@thewilkybarkid we cannot control what the size of the image is that is referenced. The hypothes.is client will attempt to render it regardless.

Reply:
Depending on what the design request is, this should become HTML (eg `<img>` or `<a>`). Maybe one day we can integrate Hypothesis and IIIF...

Reply:
At the moment the designs are showing it displayed as a text link `<a>`.